docker/kubernetes/sawtooth-kubernetes-default-poet.yaml

```diff
@@ -17,23 +17,46 @@ items:
       spec:
         containers:
           - name: sawtooth-intkey-tp-python
-            image: hyperledger/sawtooth-intkey-tp-python:1.0
+            image: hyperledger/sawtooth-intkey-tp-python:1.1
             command:
               - bash
             args:
               - -c
               - "intkey-tp-python -vv -C tcp://$HOSTNAME:4004"
 
+          - name: sawtooth-poet-engine
+            image: hyperledger/sawtooth-poet-engine
+            command:
+              - bash
+            args:
+              - -c
+              - "if [ ! -f /poet-shared/poet-enclave-measurement ]; then \
+                    poet enclave measurement >> /poet-shared/poet-enclave-measurement; \
+                fi && \
+                if [ ! -f /poet-shared/poet-enclave-basename ]; then \
+                    poet enclave basename >> /poet-shared/poet-enclave-basename; \
+                fi && \
+                if [ ! -f /poet-shared/simulator_rk_pub.pem ]; then \
+                    cp /etc/sawtooth/simulator_rk_pub.pem /poet-shared; \
+                fi && \
+                while [ ! -f /poet-shared/validator-0/keys/validator.priv ]; do sleep 1; done && \
```

Comment:
With these sleep loops, should it print out when it sleeps for debugging purposes?